doc/getting-started/overview_dev.rst

```diff
@@ -282,8 +310,11 @@ then boot ACRN.
 
 At a high level, you will:
 
-* Copy the built ACRN hypervisor files, kernel files, and launch scripts from
-  the development computer to the target.
+* Copy the built ACRN hypervisor files, Service VM kernel files, and launch
+  scripts from the development computer to the target. The Service VM kernel
+  files replace parts of the Ubuntu installation we installed and used for
+  running the Board Inspector, with the Linux kernel we built based on the board
```

Comment:
I think this is a bit misleading to the extent there is nothing in the Linux kernel we provide that is taken from either the board or scenario XML.

Reply:
@gvancuts I thought we're making some tweaks to the linux kernel based on our configuration.  Is the make of the acrn-kernel not dependent on any of the scenario or hypervisor configuration options?

Reply:
@dbkinder Ok, I removed it in the latest commit.